federated-mongodb/README.md

```diff
@@ -78,28 +94,29 @@ Now that the `kubefedctl` binary has been acquired the next step is joining the
 Using the `kubeconfig` file that was generated, verify the Operator has been successfully deployed.
 ~~~sh
 $ oc get csv -n federated-mongo
```

Comment:
Instead of checking the operator, you should check the KubeFed controller pods. You could get the deployment `kubefed-controller-manager` status to ensure KubeFed controller is running.